sno/geometry.py

```diff
@@ -1,11 +1,34 @@
 import binascii
 import json
 import math
+import re
 import struct
 
 from osgeo import ogr, osr
 
 
+_epsg_re = re.compile(r'^(?:epsg:)?(\d+)$', re.IGNORECASE)
+
+
+def make_srs(srs_text):
+    """
+    Creates an OGR SpatialReference object from the given string.
+    Accepted input is either WKT or 'EPSG:<id>'
```

Comment:
Should we use [`SetFromUserInput()`](https://gdal.org/api/ogrspatialref.html?highlight=setfromuserinput#_CPPv4N19OGRSpatialReference16SetFromUserInputEPKc) here?